Source/RevitIFCTools/IFCEntityListWin.xaml.cs

```diff
@@ -435,7 +436,8 @@ List<IFCPropertySetDef> DefineFXProperties ()
          props.Add("ProjectDevelopmentType");
          props.Add("Project Location");
          props.Add("System");
-         fxPSets.Add(pset);
+         pset.Properties = props;
+         //fxPSets.Add(pset);
```

Comment:
Cleanup/delete lines you are not using nor you are intended to use them later. Lear to do proper housekeeping and not to leave litters all over the place